data/levelsData.js

```diff
@@ -1,33 +1,30 @@
-import answersData from 'data/answersData';
-import 'core-js/actual/array/group-by';
+import answersData from "data/answersData";
+import "core-js/actual/array/group-by";
 
-const levelsData = answersData;
+let levelsData = answersData;
 // splits url into sections and levels
-const LevelTitles = levelsData.map(
-  (a) =>
-    a.url
-      .split('/')
-      .filter((element) => {
-        return (
-          element !== '' &&
-          element !== 'answers'
-        );
-      })
-);
+const LevelTitles = levelsData.map((a) => {
+  return a.url.split("/").filter((element) => {
+    return element !== "" && element !== "answers";
+  });
+});
 
 // adds section, tier, tierLevel to object
-levelsData.map((item, index) => {
-  item.section = LevelTitles[index]
-    ? LevelTitles[index][0]
-    : '';
-  item.tier = LevelTitles[index]
-    ? LevelTitles[index][1]
-    : '';
-  item.tierLevel = LevelTitles[index]
-    ? LevelTitles[index][2]
-    : '';
-});
+levelsData = levelsData
+  .map((item, index) => {
+    if (item.url === "") {
+      return null;
+    }
+    return {
+      ...item,
+      section: LevelTitles[index] ? LevelTitles[index][0] : "",
+      tier: LevelTitles[index] ? LevelTitles[index][1] : "",
+      tierLevel: LevelTitles[index] ? LevelTitles[index][2] : "",
+    };
+  })
+  .filter((ld) => ld !== null);
```

Comment:
if you return inside a map, it should have a container/variable to place into, look at line13, it should be `levelsData = levelsData.map...`

so inside map function, either we return null (l16) or return the whole item including the new props (section,tier,tierlevel) lines 19-22